.github/ISSUE_TEMPLATE/dataset-request.md

```diff
@@ -0,0 +1,28 @@
+---
+name: Dataset request
+about: Suggest an idea for a new dataset to be published on ADX
+title: ''
+labels: ''
+assignees: ''
+
+---
+
```

Comment:
Extra line breaks are needed in the md file for GitHub to render the line breaks properly, e.g:
```
**Dataset name | Publisher**

The name of the dataset and publisher

Ex. COVID-19 Data in the United States | The New York Times

**Link to the dataset**

A link to where the dataset can be sourced from

Ex. https://github.com/nytimes/covid-19-data

**License information (if available)**

License information, as we can generally only publish free and open data

Ex. https://raw.githubusercontent.com/nytimes/covid-19-data/master/LICENSE

**Frequency (if available)**

How often the dataset is published

Ex. Daily

**Formats (if available)**

What formats the data is published or requested to be in

Ex. CSV, JSON
```

Reply:
Thanks Derek, incorporated this change.